ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -631,7 +669,8 @@ private void prepareForSending() {
     void onFillComplete() {
       buf.flip();
       prepareForSending();
-      chunkReadyTimeMs = time.milliseconds();
+      chunkReadyAtMs = time.milliseconds();
+      routerMetrics.chunkFillTimeMs.update(chunkReadyAtMs - chunkFreeAtMs);
```

Comment:
Are metadata chunk and data chunk treated in the same way, so it is a single metric that reports `chunkFillTime` for both chunk types?